### DISCUSSION TITLE
Window can no longer be resized below some minimum width anymore

### DISCUSSION THREAD

**Discussion by sturgman** [sumatrapdfreader/sumatrapdf#2424] [General]
Not sure which update changed this behavior, but I can no longer resize the SumatraPDF window below some (seemingly arbitrary) minimum width. I relied on being able to change the width of the SumatraPDF window to any width needed for a particular application. Was this change purposeful? Can it be toggled? Thanks in advance.

**Comment by kjk** [comment]
This is on purpose and cannot be turned off.
